Dockerfile

```diff
@@ -18,6 +18,8 @@ COPY . .
 
 # Install minimum necessary dependencies, build Cosmos SDK, remove packages
 RUN apk add --no-cache $PACKAGES && \
+  git submodule init && \
+  git submodule update && \
```

Comment:
this commands can be one
git submodule update --init

Reply:
fixed